configs/common/packages.yaml

```diff
@@ -58,6 +58,8 @@
     esmf:
       version: ['8.4.2']
       variants: ~xerces ~pnetcdf snapshot=none ~shared +external-parallelio
+      require:
```

Comment:
why for parallelio, I thought this was needed for ESMF?

Reply:
This is for esmf

Reply:
Argh ... stupid line break is playing tricks on me, made the parallelio appear as a newline. Sorry!